docs/tracing/docs/030-trace-comparison.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Trace Comparison
+---
+
+Trace comparison allows the structure of two traces to be compared. Each trace is rendered as a tree of connected services and operations. The comparison differences between two traces are highlighted with color-coding.
+
+* In the search page for traces, select the traces to compare and then press the `Compare Traces` button.
+
+![Tracing architecture](assets/list-traces.png)
+
+* In the comparison page, the first two traces (from the previous step) will be compared together. The first trace is called `A` and the second trace is called `B`.
+
+![Tracing architecture](assets/compare-traces-1.png)
+
+* There is an option to select which traces to compare from the top menus for `A` and `B`.
+
+![Tracing architecture](assets/compare-traces-2.png)
+
+* Trace spans have different colors which indicate different meaning:
+  * `Dark` colors indicate that the span is missing from one of the traces:
+    * `Dark red`: The span is only in trace `A`.
+    * `Dark green`: The span is only in trace `B`.
+  * `Light` colors indicate that the span is present in both traces but occurs more in one of the traces:
+    * `Light red`: The span in A has more spans than `B`.
```

Comment:
1. `Light red: The span in **A** has more spans than **B**.`

2. Just to be sure: 
A span can have more additional spans?

Reply:
Regarding point 2, the answer is yes, as per Jaeger blog post for this feature.